vars/stageStatusFilename.groovy

```diff
@@ -0,0 +1,19 @@
+// vars/stageStatusFilename.groovy
+
+import java.net.URLEncoder
+
+@NonCPS
+String urlEncode() {
+    return URLEncoder.encode(env.STAGE_NAME, 'UTF-8')
+}
```

Comment:
Is a function really necessary here for a single line of un-reused code?  Couldn't this just be done in-place in the `call()`?

Reply:
I thought that the @NonCPS code needed to be in its own function not inline code.
Documentation refers to steps as being CPS transformed and methods can be annotated as @NonCPS.

Reply:
I have not really looked into what all of that [Non]CPS stuff is TBH.  Probably should at some point.  But I wonder if it really matters for one-liner.

Reply:
Updated documentation for the function in the next push.